apps/web/components/shared/trendingTopics/TrendingTopics.tsx

```diff
@@ -1,38 +1,58 @@
+'use client';
+
 import { Tag } from 'ui/core';
 import { TrendingTopicsSkeleton } from './TrendingTopicsSkeleton';
 import { TrendingTopicsProps } from './TrendingTopics.types';
+import { defaultLocale } from 'intl';
```

Comment:
Merge both imports in the same line
```suggestion
import { defaultLocale, LocaleProps } from 'intl';
```